src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -863,7 +863,7 @@ HRESULT STDMETHODCALLTYPE CorProfiler::JITCompilationStarted(FunctionID function
 
     // The first time a method is JIT compiled in an AppDomain, insert our startup
     // hook which, at a minimum, must add an AssemblyResolve event so we can find
-    // Datadog.Trace.ClrProfiler.Managed.dll and its dependencies on-disk since it
+    // Datadog.Trace.dll and its dependencies on-disk since it
     // is no longer provided in a NuGet package
```

Comment:
For context, at a previous point in time the user had to do two things to get automatic instrumentation working:
1. Install the platform package
2. Add the Datadog.Trace.ClrProfiler.Managed NuGet package

At a certain point we added the new loader mechanisms so that only the platform package was required. So the code path here is conveying that we must ensure the loader was injected, which will trigger the Datadog.Trace.ClrProfiler.Managed assembly load and tell us if we can proceed with instrumentation or not

Reply:
Thanks. I'll fix the comment.

Reply:
Removed the "no longer provided in a NuGet package" in 2a4b753ead784999d7735111ff3d9be7eb63b409 since that was referring to `Datadog.Trace.ClrProfiler.Managed.dll`